wagtail/admin/templates/wagtailadmin/account/settings_panels/base.html

```diff
@@ -1,5 +1,7 @@
 {% block content %}
-    {% for field in form %}
-        {% include "wagtailadmin/shared/field.html" with field=field %}
-    {% endfor %}
+    <div class="settings-panel w-mt-5">
+        {% for field in form %}
+            {% include "wagtailadmin/shared/field.html" with field=field %}
+        {% endfor %}
+    </div>
 {% endblock %}
```

Comment:
Just a drive by suggestion, maybe we should put this wrapper div on the outside of `content`. Otherwise any customisations extending this template would also have to include this div wrapper.

```html
<div class="w-account-settings-panel w-mt-5">
    {% block content %}
        {% for field in form %}
            {% include "wagtailadmin/shared/field.html" with field=field %}
        {% endfor %}
    {% endblock %}
</div>
```

Additionally, for new HTML we are trying to prefix custom styles with `w-`.

Reply:
Oh yeah, I see what you mean, to give the flexibility when customizing make the base.html
Thanks